### PR TITLE
schema migration

### PR DESCRIPTION
**Pull request by jiqiang90**
# Description

Current implementation we only allow user to add new entity in schema while project already in syncing, and modification to existing entity will be ignored. This change will allow user to : 

- Add new field to entity
- Drop current filed on entity  (this will also delete corresponding column in table and data)
- Change current field attributes (This will be limited from future)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Checklist

- [x] I have tested locally
- [x] I have performed a self review of my changes
- [ ] Updated any relevant documentation
- [ ] Linked to any relevant issues
- [ ] I have added tests relevant to my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] My code is up to date with the base branch

### CODE REVIEW

**Comment by jamesbayly** [comment]
Jesus, what other one line feature requests can we make?

**Comment by jamesbayly** [comment]
Jesus, what other one line feature requests can we make?

File: packages/node/src/indexer/store.service.ts [hunk 1]
```diff
@@ -303,7 +303,9 @@ export class StoreService {
     }
     this.metaDataRepo = MetadataFactory(this.sequelize, schema);
 
-    await this.sequelize.sync();
+    // this will allow alter current entity, including fields
+    // TODO, add rules for changes, eg only allow add nullable field
```
Comment (stwiname) [inline packages/node/src/indexer/store.service.ts:307]: Is this going to happen in another PR?

**Review by stwiname — COMMENTED** [review]
How does a user use this feature? They're going to want it to happen at a certain block right?

**Review by stwiname — APPROVED** [review]

**Pull request merged**